apps/fluent-tester/src/E2E/RadioGroupExperimental/pages/RadioGroupExperimentalTabsPageObject.win.ts

```diff
@@ -0,0 +1 @@
+// TODO
```

Comment:
Do you plan to do this in the next PR?

Reply:
Yes, this will be in the next PR aligning with the new async testing framework Sammy is working on.